### PR TITLE
Decompile unk_02007200 -> systask_with_env

### PR DESCRIPTION
**Pull request by tgsm**
Short and sweet.

### CODE REVIEW

File: src/systask_with_env.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#include "global.h"
```
Comment (adrienntindall) [inline src/systask_with_env.c:1]: systask_with_env sounds too wordy but I cant think of a better one so as long as @red031000 doesn't have an issue with it this file looks good to me

Comment (red031000) [reply src/systask_with_env.c:1]: systask_environment.c imo
and change Env to Environment generally, would keep it fairly consistent with others

**Review by red031000 — CHANGES_REQUESTED** [review]

**Review by red031000 — APPROVED** [review]
